src/zos_files/zowe/zos_files_for_zowe_sdk/file_system.py

```diff
@@ -122,19 +150,24 @@ def unmount(self, file_system_name, options={}, encoding=_ZOWE_FILES_DEFAULT_ENC
         response_json = self.request_handler.perform_request("PUT", custom_args, expected_code=[204])
         return response_json
 
-    def list(self, file_path_name=None, file_system_name=None):
+    def list(self, file_path_name: Optional[str] = None, file_system_name: Optional[str] = None) -> dict:
         """
-        list all mounted filesystems, or the specific filesystem mounted at a given path, or the
+        List all mounted filesystems.
+
+        It could also list the specific filesystem mounted at a given path, or the
         filesystem with a given Filesystem name.
 
-        Parameter
-        ---------
-        file_path: str - the UNIX directory that contains the files and directories to be listed.
-        file_system_name: str - the name for the file system to be listed
+        Parameters
+        ----------
+        file_path_name: Optional[str]
+            the UNIX directory that contains the files and directories to be listed.
+        file_system_name: Optional[str]
+            the name for the file system to be listed
```

Comment:
Same comment as above about sentence case for consistency
```suggestion
        file_path_name: Optional[str]
            USS directory that contains the files and directories to be listed
        file_system_name: Optional[str]
            Name of the file system to be listed
```